README.md

```diff
@@ -158,43 +158,68 @@ Setting up and configuring of all these protections depends on and requires a lo
 
 ## Set up development environment
 
-### Local setup
+### Developing dependencies
+
+- API
+  - Required packages:
+    - `golang` >= 1.22
+    - `libexif` >= 1.15.1
+    - `dlib`
+    - `libjpeg`
+    - `libblas` 
+    - `libcblas`, recommending using `libatlas-base`
+    - `liblapack`
+  - Optional tools during developing:
+    - `reflex`
+    - `sqlite`
+- UI
+  - Required packages:
+    - `node` >= 18
+
+In Debian/Ubuntu, install dependencies:
 
-1. Install a local mysql server, and make a new database
-2. Rename `/api/example.env` to `.env` and update the `MYSQL_URL` field
-3. Rename `/ui/example.env` to `.env`
+```sh
+$ sudo apt install golang libexif-dev libdlib-dev libjpeg-dev libblas-dev libatlas-base-dev liblapack-dev libdlib-dev # For API requirement
+$ sudo apt install reflex sqlite3 # For API optional tools
+$ sudo apt install node # For UI
+```
 
-### Start API server
+In macOS, install dependencies:
 
-Make sure [golang](https://golang.org/) is installed.
+```sh
+$ brew install golang libheif dlib jpeg # For API
+$ brew install reflex sqlite3 # For API optional tools
+$ brew install node # For UI
+```
 
-Some C libraries are needed to compile the API, see [go-face requirements](https://github.com/Kagami/go-face#requirements) for more details.
-They can be installed as shown below:
+Please follow the package manager guidance if you don't use `apt` or `homebrew`.
 
-```sh
-# Ubuntu
-sudo add-apt-repository ppa:strukturag/libheif
-sudo add-apt-repository ppa:strukturag/libde265
-sudo apt-get update
-sudo apt-get install libdlib-dev libblas-dev libatlas-base-dev liblapack-dev libjpeg-turbo8-dev libheif-dev
-# Debian
-sudo apt-get install libdlib-dev libblas-dev libatlas-base-dev liblapack-dev libjpeg62-turbo-dev libheif-dev
-# macOS
-brew install dlib libheif
+### Local setup
 
-```
+1. Rename `/api/example.env` to `.env`
+  - Update `PHOTOVIEW_SQLITE_PATH` if you don't want to put sqlite file under `/api`
```

Comment:
... or set a different DBMS driver, followed by commenting our the SQLite path variable and uncommenting the corresponding connection string variable for the new driver (and update its value to correspond to your setup)

Reply:
Done